### PR TITLE
Privacy pro Onboarding promo June

### PR DESCRIPTION
**Pull request by cmonfortep**
Asana: https://app.asana.com/0/0/1207546055898908/f

Test:
- Check requirements in https://app.asana.com/0/0/1207546055898908/f
- Use staging endpoint (you can enable it in internal RMF settings)
- Ensure copies are correct (validate config)
- Ensure URLs are correct (validate config)
- For testing, you will need to hardcode some values in the codebase (e.g: ppro eligible to return true, days since installed 3)
- Ensure a user will see at least 1 message but only 1 (if you interact with one, you won't see another)

### CODE REVIEW

File: live/android-config/android-config.json [hunk 1]
```diff
@@ -33,39 +96,114 @@
...
+        },
+        "daysSinceInstalled": {
+          "value": 3
         }
       }
     },
     {
       "id": 2,
+      "targetPercentile": {
+          "before": 0.33
```
Comment (nalcalag) [inline live/android-config/android-config.json:117]: shouldn't be this 0.25?

File: live/android-config/android-config.json [hunk 2]
```diff
@@ -33,39 +96,114 @@
...
+        },
+        "daysSinceInstalled": {
+          "value": 3
+        }
+      }
+    },
+    {
+      "id": 3,
+      "targetPercentile": {
+          "before": 0.5
```
Comment (nalcalag) [inline live/android-config/android-config.json:145]: ditto

File: live/android-config/android-config.json [hunk 3]
```diff
@@ -33,39 +96,114 @@
...
+        },
+        "daysSinceInstalled": {
+          "value": 3
+        }
+      }
+    },
+    {
+      "id": 4,
+      "targetPercentile": {
+          "before": 1
```
Comment (nalcalag) [inline live/android-config/android-config.json:173]:
```suggestion
          "before": 0.25
```

**Comment by nalcalag** [comment]
@cmonfortep not sure how `targetPercentile` works in the new RMF, but in the old surveys framework you just needed to add the percentile per variant which here would be 0.25 each.
In that case, because all messages share matchingRules, could we just create 1 and assign to all of them?

**Comment by cmonfortep** [comment]
Posted in Asana, but also posting here. Since the goal is to have 25% chances to see one of the 4 messages, and they share target audience, we need a different value on each percentile. Using the same percentile will give more probability to appear to the first one, since we evaluate in order.

**Review by nalcalag — APPROVED** [review]
Thanks for the percentile explanation. LGTM!

**Pull request merged**